lib/flop.ex

```diff
@@ -1959,17 +1975,24 @@ defmodule Flop do
       iex> push_order(%Flop{order_by: [:id], before: "DEF"}, :name)
       %Flop{order_by: [:name, :id], order_directions: [:asc], before: nil}
   """
-  @spec push_order(Flop.t(), atom | String.t()) :: Flop.t()
+  @spec push_order(Flop.t(), atom | String.t(), [option()]) :: Flop.t()
```

Comment:
I'd like to reserve the `option` type for the query functions. This function only accepts a single option.

```suggestion
  @spec push_order(Flop.t(), atom | String.t(), keyword) :: Flop.t()
```